documentation/book/faq.adoc

```diff
@@ -1,9 +1,52 @@
-[appendix]
-== Frequently Asked Questions
 
-=== Cluster Operator
+= Frequently Asked Questions
 
-==== Log contains warnings about failing to acquire lock
+== Cluster Operator
+
+=== Why does {ProductName} cluster admin privileges to install?
+
+When installing {ProductName} you need to have the ability to create Custom Resource Definitions (CRDs). 
+These are the resources that tell {PlatformProductName} about the {ProductName}-specific resources, such as `Kafka`, `KafkaConnect` and so on.
+Because CRDs are a cluster-scoped resource (rather than being scoped to a particular {PlatformProductName} namespace) typically cluster admin privileges are required to be able to do this.
+
+Installation of {ProductName} also requires the ability to create some `ClusterRoles` and `ClusterRoleBindings`, similarly to CRDs these are cluster-scoped resources and so, again, typically cluster admin privileges are required to be able to do this.
+
+The cluster administrator can inspect all the resources being installed (in the `/install/` directory) to assure themselves that the `ClusterRoles` do not grant unnecessary privileges. For more information about why the Cluster Operator installation resources grant the ability to create `ClusterRoleBindings` see the following question.
+
+After installation, the Cluster Operator will run as a regular `Deployment`; any non-admin user with privileges to access the `Deployment` can configure it.
+
+[id='normal-user-access-custom-resources-{context}']
+By default, normal users will not have the privileges necessary to manipulate the custom resources, such as `Kafka`, `KafkaConnect` and so on, which the Cluster Operator deals with.
+These privileges can be granted using normal RBAC resources by the cluster administrator. See xref:assembly-getting-started-strimzi-admin-str[this procedure] for more details of how to do this.
+
+
+=== Why does the Cluster Operator require the ability to create `ClusterRoleBindings`? Is that not a security risk?
+
+{PlatformProductName} has built-in link:https://kubernetes.io/docs/reference/access-authn-authz/rbac/#privilege-escalation-prevention-and-bootstrapping[privilege escalation prevention^]. 
+That means that the Cluster Operator cannot grant privileges it does not have itself. 
+Which in turn means that the Cluster Operator needs to have the privileges necessary for _all_ the components it orchestrates.
+
+In the context of this question there are two places where the Cluster Operator needs to create bindings to `ClusterRoleBindings` to `ServiceAccounts`:
```

Comment:
_Suggested change:_

In the context of this question, the Cluster Operator creates bindings to `ClusterRoleBindings` and `ServiceAccounts` in two places.